content/news/036/index.md

```diff
@@ -113,7 +113,7 @@ Some of the new features include:
 - A better data model for meshes, based on groups and channels.
 - Game engine integration with Godot, more engines coming soon.
 - Introduce Lua as an extension language.
-- Add *many* new nodes: Extrude along curve, Copy to points...
+- Add _many_ new nodes: Extrude along curve, Copy to points...
```

Comment:
Looks like you have a linter that is touching lines that aren't yours, can you revert them?